src/at_chops/aes_ctr.c

```diff
@@ -0,0 +1,173 @@
+#ifdef BUILD_MBEDTLS
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <mbedtls/aes.h>
+#include "aes_ctr.h"
+#include "base64.h"
+
+#define AES_KEY_BITS 256
```

Comment:
How difficult is it to retrieve the key length from the key passed into the function? If we can do that, then we can support CTR for any length out of the box.

Reply:
Done in https://github.com/atsign-foundation/at_c/pull/7/commits/b09d205d4afb7c078c8af7d1f3a898ffceb57e58

Reply:
But key size can be derived from the key, can't it?

Reply:
The keys are encoded in Base64, so that would probably involve decoding it first which is I think an unnecessary operation for now.

Reply:
Noted